tests/test_config_parsing/test_forward_model.py

```diff
@@ -139,19 +139,22 @@ def test_that_comments_are_ignored():
 
 @pytest.mark.usefixtures("use_tmpdir")
 def test_that_quotations_in_forward_model_arglist_are_handled_correctly():
-    """This is a regression test, making sure that string behave consistently
+    """This is a regression test, making sure that strings behave consistently.
     The previous fail cases are described in the comments of the config. They
-     should all result in the same
-     See https://github.com/equinor/ert/issues/2766"""
+    should all result in the same.
+    See https://github.com/equinor/ert/issues/2766"""
 
     test_config_file_name = "test.ert"
     test_config_contents = dedent(
         """
-        NUM_REALIZATIONS  1
-        FORWARD_MODEL COPY_FILE(<FROM>='some, thing', <TO>="some stuff", <FILE>=file.txt) -- success
-        FORWARD_MODEL COPY_FILE(<FROM>='some, thing', <TO>='some stuff', <FILE>=file.txt) -- some stuff becomes somestuff
-        FORWARD_MODEL COPY_FILE(<FROM>="some, thing", <TO>="some stuff", <FILE>=file.txt) -- util abort
-        """  # noqa: E501
+    NUM_REALIZATIONS  1
+    -- success:
+    FORWARD_MODEL COPY_FILE(<FROM>='some, thing', <TO>="some stuff", <FILE>=file.txt)
+    -- some stuff becomes somestuff:
```

Comment:
Comments at the end of the line was part of the test here

Reply:
like it was depending on it? argh

Reply:
based on the docstring of the test function i though it was just meta information conveying past behaviour - it still looks like that to me on further inspection. am i missing something?

Reply:
I guess some additional comments are warranted. The FORWARD_MODEL line is "special" in that it permits "--" in named inside the parenthesis, but outside it is a comment.

Reply:
I think we can remove those comments in this test then

Reply:
we can find the old behaviour documented in at least one issue, as far as i remember - great, will do that!

Reply:
pushed